examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,1018 @@
+     ]
+    }
+   ],
+   "source": [
+    "from autoevals import (\n",
+    "    ContextEntityRecall,\n",
+    "    ContextRelevancy,\n",
+    "    ContextRecall,\n",
+    "    Faithfulness,\n",
+    "    AnswerCorrectness,\n",
+    ")\n",
+    "from braintrust import Eval\n",
+    "\n",
+    "# TODO?: Could put some effort into finding a way to reduce this boilerplate code, which I think can be really confusing to a new braintrust user\n",
+    "\n",
+    "\n",
+    "async def context_entities(input, output, expected, metadata):\n",
+    "    return await ContextEntityRecall().eval_async(\n",
+    "        output=output, expected=expected, context=metadata[\"context\"]\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "async def context_relevancy(input, output, expected, metadata):\n",
+    "    return await ContextRelevancy().eval_async(\n",
+    "        input=input,\n",
+    "        output=output,\n",
+    "        expected=expected,\n",
+    "        context=metadata[\"context\"],\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "async def context_recall(input, output, expected, metadata):\n",
+    "    return await ContextRecall().eval_async(\n",
+    "        input=input,\n",
+    "        output=output,\n",
+    "        expected=expected,\n",
+    "        context=metadata[\"context\"],\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "async def faithfulness(input, output, expected, metadata):\n",
+    "    return await Faithfulness().eval_async(\n",
+    "        input=input,\n",
+    "        output=output,\n",
+    "        expected=expected,\n",
+    "        context=metadata[\"context\"],\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "async def answer_correctness(input, output, expected, metadata):\n",
+    "    return await AnswerCorrectness().eval_async(\n",
+    "        input=input,\n",
+    "        output=output,\n",
+    "        expected=expected,\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "result = await Eval(\n",
+    "    name=\"Ragas Retrieval\",\n",
+    "    experiment_name=f\"{RELEVANCE_MODEL} picks {TOP_K_TO_USE} out of {TOP_K_TO_FETCH}, answer w {QA_ANSWER_MODEL}\",\n",
+    "    data=[\n",
+    "        {\n",
+    "            \"input\": x[\"input\"],\n",
+    "            \"expected\": x[\"expected\"],\n",
+    "            \"metadata\": {\"context\": x[\"context\"], \"n_contexts\": x[\"n_contexts\"]},\n",
+    "        }\n",
+    "        for x in rag_results_w_context[:NUM_QA_PAIRS]\n",
+    "    ],\n",
+    "    task=lambda input: q_and_a_dict[input],\n",
+    "    scores=[\n",
+    "        answer_correctness,\n",
+    "        context_entities,\n",
+    "        context_relevancy,\n",
+    "        context_recall,\n",
+    "        faithfulness,\n",
+    "    ],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using RAGAS and Braintrust to track improvements\n",
```

Comment:
maybe useful to have a screenshot above this and include a quick (somewhat repetitive, but that's ok) takeaway with the next paragraph